app/scripts/views/settings.js

```diff
@@ -206,6 +206,7 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
         })
         .fin(function () {
           self.logViewEvent('signout.success');
+          self.relier.unset('uid');
```

Comment:
Can you leave a note here about why this is needed, otherwise we are going to forget in 6 months. :/